toolchain/check/impl_validation.cpp

```diff
@@ -370,8 +370,20 @@ static auto ImportFinalImplsWithImplInFile(Context& context) -> void {
     if (!import_ir_id.has_value()) {
       continue;
     }
+    auto interface_owning_decl_id = interface.first_owning_decl_id;
+    if (!interface_owning_decl_id.has_value()) {
+      continue;
+    }
```

Comment:
If `first_owning_decl_id` was None, the `import_ir_id` returned from GetIRId would be None, and we would have returned above, no?